engine/src/flutter/lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -443,6 +444,12 @@ enum EngineSemanticsRole {
   /// A cell in a [table] contains header information for a column.
   columnHeader,
 
+  /// A container that display its children in a list layout.
```

Comment:
nit: s/display/displays/

Instead of saying "list layout" (how is a list laid out? can it be horizontal?), consider saying "...whose children are logically a list of items". We should also probably explain that it is expected that each child node has the role `[listItem]`.